packages/testcontainers/src/docker.ts

```diff
@@ -0,0 +1,192 @@
+import Dockerode, {DockerOptions, Container} from 'dockerode'
+import fetch from 'node-fetch'
+import JSONBig from 'json-bigint'
+
+export type Network = 'mainnet' | 'testnet' | 'regtest'
```

Comment:
ex. here, I think you have defined it as well on another file, might be good to reuse this